omod/src/main/compass/sass/home.scss

```diff
@@ -16,4 +16,56 @@ hr.separator {
     padding: 10px 0;
     border-top: 1px dashed #CCC;
   }
-}
\ No newline at end of file
+}
+
+.homeList a  {
+  font-size: unset;
+  white-space: normal;
+  width: 166px !important;
+  color: #363463;
+}
+
+.homeList a:hover{
```

Comment:
these can go under nested styles in scss.

Reply:
done